cypress/support/pages/homeСatalogue.pageObject.js

```diff
@@ -17,6 +17,18 @@ class HomeAndCataloguePageObject extends PageObject {
     cy.contains('.hrefch', product)
       .click();
   }
+  
+   addToCart() {
+    cy.contains('Add to cart').click();
+    cy.on('window:alert', (str) => {
+      expect(str).to.equal('Product added');
+    });
+  }
+
+  clickOnCart() {
```

Comment:
The function name 'clickOnCart' is not a proper camelCase. Consider renaming it to 'clickOnCartButton' for better clarity.